test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -248,7 +247,7 @@ def test_successful_asynchronous_adjust
     assert_successful_response(authorize)
     assert adjust = @gateway_latam.adjust(@amount * 2, authorize.authorization, @options)
     assert_success adjust
-    assert capture = @gateway_latam.capture(@amount, authorize.authorization, @options)
+    assert capture = @gateway_latam.capture(@amount, authorize.authorization, @options.merge({ national_tax_indicator: 1 }))
```

Comment:
Is there a reason why this value is removed from the `@options` hash only to be merged back into the hash in all of these tests?

Reply:
To keep the test cases same, this value is being passed in the options as previously we were sending `national_tax_indicator ` in **@options** and the otherTax values were being set for capture only. In this implementation, this indicator key is being sent in for auth and purchase calls as well so this key was removed from global @options and are being passed for capture by merge.